src/Core/System/Snippet/SnippetPatterns.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Snippet;
+
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[Package('discovery')]
+abstract class SnippetPatterns
```

Comment:
why an abstract class? imo it should be final and have a private constructor, so it cannot be instantiated. 

the `SnippetValidatorInterface` might also be a good place for the constants

Reply:
Ah, okay. True.